authapi/urls.py

```diff
@@ -40,4 +40,7 @@
 urlpatterns = [
     url(r'^', include(router.urls)),
     url(r'^/user/tokens/$', views.TokenView.as_view(), name='create-token'),
+    url(
+        r'^/user/permissions/$', views.UserPermissionsView.as_view(),
+        name='get-user-permissions')
```

Comment:
Ah right, yes we did. We just haven't updated the docs yet. Let me update it and change this.